src/recommendations/data/reducers.js

```diff
@@ -0,0 +1,27 @@
+import { USER_RECOMMENDATIONS } from './actions';
+
+export const defaultState = {
+  recommendations: [],
+  isLoading: true,
```

Comment:
can we use the approach for API status that is being used in the common components reducer for thirdPartyAuthContext?

Reply:
Actually we can remove this state. We are async calling algolia and on progressive profiling page we are already handling loading state.
Whats your thought on it?

Reply:
lets call on this.

Reply:
how is it linked to the loading state of progressive profiling page?